style/mainGrid.css

```diff
@@ -0,0 +1,84 @@
+html {
+    box-sizing: border-box;
+}
+
+*,
+*:before.
+*:after {
+    box-sizing: inherit;
+}
+
+.grid-container {
+    max-width: 1em;
+    margin: auto;
+    padding: auto;
+}
+
+.top-row {
+    width: 100%;
+    height: 33.33333%;
```

Comment:
I think you might want to use `vh` units for height fields. So this should be `height: calc(100vh / 3)`

`1vh` = 1% of the height of the browser.